base/missing.jl

```diff
@@ -188,3 +187,30 @@ function Base.iterate(itr::SkipMissing, state...)
     end
     item, state
 end
+
+"""
+    coalesce(x, y...)
+
+Return the first value in the arguments which is not equal to [`missing`](@ref),
+if any. Otherwise return `missing`.
+
+# Examples
+
+```jldoctest
+julia> coalesce(missing, 1)
+1
+
+julia> coalesce(1, missing)
+1
+
+julia> coalesce(nothing, 1)  # returns `nothing`
```

Comment:
Probably better remove this example, there's no reason to think that `nothing` is treated differently from any other value.

Reply:
I think the point of the example is to demonstrate that.

Reply:
I think the point of the example is to demonstrate that.

Reply:
Yes, I understand that, but my point is that nobody would have imagined `coalesce` would consider `nothing` as a missing value, except for the fact that it was the previous behavior. And since most people won't know that history...

Anyway, not a big deal.